### PR TITLE
Update for agama auto installation

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket: https://progress.opensuse.org/issues/163943
- Needles: N/A
- Verification run: 
  https://openqa.opensuse.org/tests/4407119#step/auto/6  x86_64
  https://openqa.opensuse.org/tests/4405327#step/auto/5  aarch64

### CODE REVIEW

File: tests/yam/agama/auto.pm [hunk 1]
```diff
@@ -12,11 +12,12 @@ use testapi;
 use utils 'clear_console';
 
 sub run {
-    assert_screen('agama-install-finished', 1200);
+    # workaround to add needle to trust the signing key 
+    assert_screen('agama-signing-key', 600);
+    assert_and_click('trust');
```
Comment (jknphy) [inline tests/yam/agama/auto.pm:17]: we should make this conditional, if the needle match, click and soft fail.

**Comment by jknphy** [comment]
it is not agama automation, you are referring to agama autoinstallation. Could you you apply to commit and PR title.

File: tests/yam/agama/auto.pm (hunk 1 above)
Comment (lemon-suse) [reply tests/yam/agama/auto.pm]: updated to muti-tag, please have a look.

**Comment by lemon-suse** [comment]
> it is not agama automation, you are referring to agama autoinstallation. Could you please apply to commit and PR title?

Updated.

**Pull request merged**